charts/seed-controlplane/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -83,6 +83,9 @@ spec:
         - --etcd-keyfile=/srv/kubernetes/etcd/client/tls.key
         - --etcd-servers=https://etcd-main-client:{{ .Values.etcdServicePort }}
         - --etcd-servers-overrides=/events#https://etcd-events-client:{{ .Values.etcdServicePort }}
+        {{- if .Values.enableEtcdEncryption }}
```

Comment:
Can you add `enableEtcdEncryption: false` to the `values.yaml` of this chart (contains all possible variables that may be set when rendering the chart)?

Reply:
ok